source/world_builder/utilities.cc

```diff
@@ -1220,14 +1218,16 @@ namespace WorldBuilder
 
               data_string = datastream.str();
               filesize = data_string.size();
+              WBAssertThrow(filesize < std::numeric_limits<uint64_t>::max(),
```

Comment:
This should be
```suggestion
              WBAssertThrow(filesize < std::numeric_limits<int>::max(),
```
Or not?

Reply:
hmm, filesize above is a uint64_t, but in the block below it is a filesize is an int (probably unsigned because string.size() returns an size_t). I don't think it matters much, but I will make it all an int in this block.

Reply:
well, filesize is a 64 bit value, so it is alway less than the maximum value you can store...